src/components/portrait/portrait.spec.js

```diff
@@ -134,6 +139,11 @@ describe("PortraitComponent", () => {
   });
 
   describe("Portrait styles", () => {
+    beforeEach(() => {
+      browserTypeCheck.mockImplementation(() => true);
+      isSafari.mockImplementation(() => true);
```

Comment:
This should just be in the describe block you've added targeting Safari styles really

Reply:
Decided we aren't using `isSafari`. The less we use it, the better really.